### PR TITLE
Add namespace and alias annotations to jsdoc

### PR DESCRIPTION
**Pull request by moniika**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Part of #5508

### Proposed Changes

Adds `@namespace` and `@alias` annotations.

### Reason for Changes

Adding these annotations ensures that the generated jsdoc includes the fully-qualified namespace.

### Test Coverage

Tested by running jsdoc generation script on branch.

### CODE REVIEW

File: core/renderers/common/debugger.js [hunk 1]
```diff
@@ -10,6 +10,10 @@
  */
 'use strict';
 
+/**
+ * Methods for graphically rendering a block as SVG.
```
[the remark this answers is not in the document]
Comment (gonfunko) [reply core/renderers/common/debugger.js:14]: Or from the fileoverview, which should itself probably be updated.

File: core/serialization/priorities.js [hunk 2]
```diff
@@ -6,13 +6,14 @@
```
Comment (gonfunko) [inline core/serialization/priorities.js:6]: It looks like this file was already good and the scripted changes can be reverted?

File: core/serialization/registry.js [hunk 3]
```diff
@@ -6,10 +6,15 @@
 
 /**
  * @fileoverview Contains functions registering serializers (eg blocks,
- *     variables, plugins, etc).
+ * variables, plugins, etc).
```
Comment (gonfunko) [inline core/serialization/registry.js:9]: Indentation

File: core/serialization/variables.js [hunk 4]
```diff
@@ -6,10 +6,15 @@
 
 /**
  * @fileoverview Handles serializing variables to plain JavaScript objects, only
- *     containing state.
+ * containing state.
```
Comment (gonfunko) [inline core/serialization/variables.js:9]: Indentation

File: core/serialization/workspaces.js [hunk 5]
```diff
@@ -6,10 +6,15 @@
 
 /**
  * @fileoverview Contains top-level functions for serializing workspaces to
- *     plain JavaScript objects.
+ * plain JavaScript objects.
```
Comment (gonfunko) [inline core/serialization/workspaces.js:9]: Indentation

File: core/theme/classic.js [hunk 6]
```diff
@@ -10,6 +10,11 @@
  */
```
Comment (gonfunko) [inline core/theme/classic.js:10]: Do we need to add any aliases for this and other files in core/theme?

File: core/utils/global.js [hunk 7]
```diff
@@ -10,6 +10,10 @@
  */
 'use strict';
 
+/**
+ * Provides a reference to the global object.
+ * @namespace Blockly.utils.global
+ */
 goog.module('Blockly.utils.global');
```
Comment (gonfunko) [inline core/utils/global.js:19]: Looks like we need an alias here (and maybe bring the export in line with the prevailing style)

File: core/utils/idgenerator.js [hunk 8]
```diff
@@ -11,15 +11,16 @@
...
+ * Generators for unique IDs.
+ * @namespace Blockly.utils.idGenerator
  */
 goog.module('Blockly.utils.idGenerator');
 
 
 /**
  * Namespace object for internal implementations we want to be able to
  * stub in tests.
+ * @alias Blockly.utils.idGenerator.TEST_ONLY
```
Comment (gonfunko) [inline core/utils/idgenerator.js:23]: Consider removing this so it doesn't show up in the docs

File: core/blockly.js [hunk 9]
```diff
@@ -11,7 +11,7 @@
 'use strict';
 
 /**
- * The top level namespace used to access the Blockly library.
```
Comment (gonfunko) [inline core/blockly.js:14]: This can be reverted.

File: core/blocks.js [hunk 10]
```diff
@@ -12,14 +12,15 @@
 
 /**
  * A mapping of block type names to block prototype objects.
- * @name Blockly.Blocks
+ * @namespace Blockly.blocks
```
Comment (gonfunko) [inline core/blocks.js:15]: Not quite sure what we want to do here - I think the alias should possibly be Blockly.Blocks? @cpcallen

File: core/widgetdiv.js [hunk 11]
```diff
@@ -75,6 +77,7 @@ exports.getDiv = getDiv;
 /**
  * Allows unit tests to reset the div.
  * @param {?Element} newDiv The new value for the DIV field.
+ * @alias Blockly.WidgetDiv.testOnly_setDiv
```
Comment (gonfunko) [inline core/widgetdiv.js:80]: Given that this is test only we might want to omit it?

File: core/serialization/blocks.js [hunk 12]
```diff
@@ -6,10 +6,14 @@
 
 /**
  * @fileoverview Handles serializing blocks to plain JavaScript objects only
- *     containing state.
+ * containing state.
```
[the remark this answers is not in the document]
Comment (moniika) [reply core/serialization/blocks.js:9]: `@fileoverview` is not supposed to be indented.
https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping

File: core/serialization/priorities.js (hunk 2 above)
Comment (moniika) [reply core/serialization/priorities.js:6]: Indentation is supposed to be removed. I have edited though to add back the "The top level namespace for priorities of plugin serializers." to the description.

File: core/renderers/common/debugger.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (moniika) [reply core/renderers/common/debugger.js:14]: Good catch, I've updated the description.

File: core/theme/classic.js (hunk 6 above)
Comment (moniika) [reply core/theme/classic.js:10]: Yeah, there are missing JSDoc for the themes here.
I'll address this in a follow up PR.

File: core/utils/idgenerator.js (hunk 8 above)
Comment (moniika) [reply core/utils/idgenerator.js:23]: I've added a `@ignore` tag to prevent it from showing up in the docs.

File: core/widgetdiv.js (hunk 11 above)
Comment (moniika) [reply core/widgetdiv.js:80]: I've added a `@ignore` annotation.

**Pull request closed without merging**